components/TheFooter.vue

```diff
@@ -2,10 +2,10 @@
   <footer class="footer-container section">
     <div
       class="footer-container-inner is-flex is-align-items-start is-justify-content-space-between">
-      <div class="footer-container-subs is-flex is-flex-direction-column">
-        <h4 class="subtitle is-5">
+      <section class="footer-container-subs is-flex is-flex-direction-column">
+        <h2 class="subtitle is-5">
```

Comment:
why `h2` ? some more semantic css?

Reply:
mobile lighthouse maybb?

Reply:
wait what how do you see this changes?

Reply:
I have eyes. 
Equipped with best addon 
![image](https://github.com/kodadot/nft-gallery/assets/22471030/73a8658e-38c0-4387-b3b7-f308e9c4e51d)